SPECS/mariner-repos/mariner-repos.spec

```diff
@@ -12,12 +12,13 @@ Source1:        MICROSOFT-METADATA-GPG-KEY
 Source2:        mariner-official-base.repo
 Source3:        mariner-official-update.repo
 Source4:        mariner-preview.repo
+Source5:        mariner-extras.repo
```

Comment:
It might be a good idea to merge latests changes from `1.0-dev` before applying more changes to not complicate the merge further.